### ISSUE TITLE
Demo color picker has unintuitive XYZ ranges

### ISSUE THREAD

**Issue by eeeps** [LeaVerou/color.js#341]
Forgive the “issue” (which might end up just being a request to ELI5) about [a demo](https://colorjs.io/apps/picker/); I'm not sure if there is a better place to put this.

The XYZ D50 and XYZ D65 sliders go from 0-100 even though all useful values seem to be well below 1.0.

The Absolute XYZ D65 sliders go to... much larger and stranger numbers, even though most useful values seem to be below 100.

I'm sure there is a reason for these chosen ranges. In which case, perhaps the sliders could use a logarithmic scale, in order to be able to use them to build more intuition about what these values mean and how adjusting them works?

**Comment by facelessuser** [comment]
I imagine this is just a bug. It may have been at one time they had considered scaling XYZ to 0 - 100 instead of 0 - 1, which may be why it functions as it currently does, but with current ranges, there is no reason to allow such extreme ranges.
